packages/hamburger/hamburger.scss

```diff
@@ -1,21 +1,28 @@
 @import "~@fremtind/jkl-core/mixins/_motion.scss";
 @import "~@fremtind/jkl-core/variables/_colors.scss";
 @import "~@fremtind/jkl-core/_functions.scss";
+@import "~@fremtind/jkl-core/variables/_font-size.scss";
 
 $button-focus-ring-color: $focus-color;
+$bounding-touch-size: rem(48px);
+
+$hamburger-line-width: rem(35px);
```

Comment:
For meg høres dette ut som tykkelsen på streken, ikke lengden. Kanskje endre til `$hamburger-icon-width` f.eks.? Kan jo godt ha linjetykkelsen definert også, så har du ryddet vekk alle verdier fra classedefinisjonene 😊